app/Models/Quote.php

```diff
@@ -132,4 +132,22 @@ public function service(): QuoteService
      {
          return new QuoteService($this);
      }
+
+    private function inReminderWindow($schedule_reminder, $num_days_reminder)
```

Comment:
We can probably spin this out into a trait to deduplicate the invoice/quote implementations

Reply:
I did think about this but I wasnt sure if the implementation between the 2 were exactly the same

Reply:
Would the trait be accessed from the model or the service?

Reply:
Yes, it fits for inclusion in the MakeReminders Trait